team-telnyx/webrtc
# docs(WEBRTC-245-258): add typedocs on deaf, undeaf, and toggleDeaf

- add typedocs on deaf, undeaf, and toogleDeaf

## 📝 To Do


## ✋ Manual testing

1. Navigate into `packages/js/examples/react-audio` 
2. Go to web dialer http://localhost:6006/?path=/story/webdialer--example
3. Open a web dialer and provide your SIP credentials
4. Make a call and pick up the call
5. Click on the button with the ear icon and talk to yourself and see if you can hear only one/unique voice.
6. Click again on the button with the ear icon and the icon needs to change and see if you can hear your voice twice time.

## 🦊 Browser testing

### Desktop


## 📸 Screenshots

State of hear your voice twice (remote and local stream):
![image](https://user-images.githubusercontent.com/16343871/101208683-4868a300-3651-11eb-8497-104247b6294f.png)

State of hear your voice once (only local stream):
![image](https://user-images.githubusercontent.com/16343871/101208668-4272c200-3651-11eb-8324-d4b13beae436.png)
